docarray/document/mixins/plot.py

```diff
@@ -185,15 +185,24 @@ def display_point_cloud_tensor(self) -> None:
         from IPython.display import display
         from hubble.utils.notebook import is_notebook
 
+        colors = np.tile(np.array([0, 0, 0]), (len(self.tensor), 1))
+        for chunk in self.chunks:
+            if (
+                'name' in chunk.tags.keys()
```

Comment:
Is `'name`' a key that is already established in other parts of the code / other features? If not I think I would prefer a more descriptive key here

Reply:
no sounds good, let's keep it consistent